packages/fast-components-react-msft/src/slider-label/slider-label.schema.ts

```diff
@@ -6,25 +6,30 @@ export default {
     id: "@microsoft/fast-components-react-msft/slider-label",
     formPluginId: "@microsoft/fast-components-react-base/slider-label",
     properties: {
-        maxValuePositionBinding: {
-            title: "Upper value position binding",
-            type: "string",
-            enum: [
-                "selectedRangeMin",
-                "selectedRangeMax",
-                "totalRangeMin",
-                "totalRangeMax",
+        valuePositionBinding: {
+            title: "Value position binding",
+            oneOf: [
+                {
+                    type: "number",
+                },
+                {
+                    type: "string",
+                    enum: [
+                        "selectedRangeMin",
+                        "selectedRangeMax",
+                        "totalRangeMin",
+                        "totalRangeMax",
+                    ],
+                },
             ],
         },
-        minValuePositionBinding: {
-            title: "Lower value position binding",
+        showTickmark: {
+            title: "Show tick mark",
+            type: "boolean",
```

Comment:
looking at the component, this should include `default: true`

Reply:
yep